Documentation/conf.py

```diff
@@ -86,6 +86,7 @@
 jenkins_branch = 'https://jenkins.cilium.io/view/Cilium-v' + release[0:3]
 archive_filename = archive_name +'.tar.gz'
 archive_link = 'https://github.com/cilium/cilium/archive/' + archive_filename
+archive_name = 'cilium-' + archive_name
```

Comment:
What does this do exactly?

Reply:
it adds `cilium-` prefix to the cd command here:

![Screen Shot 2019-08-05 at 5 31 16 PM](https://user-images.githubusercontent.com/25494577/62502954-dfe08c00-b7a6-11e9-8b80-06b2eb6575fa.png)

so instead of `cd master/install/kubernetes` it'll be `cd cilium-master/install/kubernetes`.

Reply:
Sorry, I missed your comment above! I just ran `make render-docs` locally and validated that your fix worked!